src/content/docs/new-relic-only/tech-writer-style-guide/processes-procedures/update-left-navigation-pane.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Update left-navigation pane
+redirects:
+  - /docs/content/style-guide/processes-and-procedures
+---
+
+Navigation for [docs.newrelic.com](https://docs.newrelic.com) is stored in `YAML` files located in the [src/nav](./src/nav) directory. Each top-level navigation should have its own configuration file. 
+
+## The configuration file
+
+Here is a snippet of the [src/nav/insights.yml](./src/nav/insights.yml) configuration. Note that the file has indentation that corresponds to the level of the navigation hierarchy. Be sure to imitate the same spacing when you make changes:
+
+```yml
+title: Insights
+children:
+  - title: Use Insights UI
+    children:
+      - title: Getting started
+        children:
+          - title: Introduction to New Relic Insights
+            path: /docs/insights/use-insights-ui/getting-started/introduction-new-relic-insights
+      - title: Manage dashboards
+        children:
+          - title: Chart types
+            path: /docs/insights/use-insights-ui/manage-dashboards/chart-types
+          - title: Filter Insights dashboards
+            path: /docs/insights/use-insights-ui/manage-dashboards/filter-insights-dashboards
+          - title: View, organize, share Insights dashboards
+            path: /docs/insights/use-insights-ui/manage-dashboards/view-organize-share-insights-dashboards
+```
+
+## Configuration file options
+
+When you're changing the navigation, keep in mind that each node in the `.yml` configuration file can have the following properties:
+
+| Key        | Required? | Description                               |
```

Comment:
This is fine, but keep in mind you can use html tables instead of markdown if you want!

Reply:
@bradleycamacho, I apologize! I hadn't pushed up the file! I only noticed that I hadn't pushed it up after your comment about the Markdown table, which I had replaced with the <Table> component. I'm must not have saved this, which is why this didn't show up in GitHub Destop as a commit to make. Anyway, could you re-read this? I made a variety of changes to it--not just the the table.

Reply:
@rhetoric101 Have you pushed up the change?